common/logger/src/lib.rs

```diff
@@ -30,7 +30,7 @@
 //! use serde_json::Value;
 //!
 //! pub struct StructuredLogEntry {
-//!     log: Option<String>,
+//!     text: Option<String>,
```

Comment:
If we're going about changing this, maybe `message` is a little more descriptive than text?

Reply:
Yeah, message is probably better.  Let me do some git magic to fix it up